Gulpfile.js

```diff
@@ -143,3 +144,48 @@ gulp.task('saucelabs', ['lint', 'build'], function (done) {
         .then(testsDone)
         .catch(testsDone);
 });
+
+gulp.task('cli', ['lint', 'build'], function (done) {
```

Comment:
How do we run this task on Travis if it requires local installed browsers :open_mouth:?